vehicle/views.py

```diff
@@ -0,0 +1,235 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import status, viewsets
+from rest_framework.decorators import api_view, action
+from rest_framework.response import Response
+from vehicle.models import Vehicle, Position, Journey
+from vehicle.serializers import VehicleSerializer, PositionSerializer, JourneySerializer
+
+
+# list(get) and create(post) for vehicle_list
+# retrieve(get), update(put), partial_update(patch) and destroy(delete) for vehicle_detail
+class VehicleViewSet(viewsets.ViewSet):
```

Comment:
@daryne77, @tudor96, please take a look to `ModelViewSet`: http://www.django-rest-framework.org/api-guide/viewsets/#modelviewset . It already has this methods.